app/models/cda/defendant.rb

```diff
@@ -1,7 +1,21 @@
 module Cda
   class Defendant < BaseModel
     def name
-      [first_name, middle_name, last_name].filter_map(&:presence).join(" ")
+      [first_name, try(:middle_name), last_name].filter_map(&:presence).join(" ")
```

Comment:
are you sure `try(:middle_name)` is needed?
the &:presence should filter it presence, correct?

Reply:
As discussed, middle_name is not nil, it's not implemented